templates/seafile-notification-server

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>seafile-notification-server</Name>
+  <Repository>seafileltd/notification-server:12.0-latest</Repository>
+  <Registry>https://hub.docker.com/r/seafileltd/notification-server/</Registry>
+  <Network>seafile-net</Network>
```

Comment:
Does this need to be on a specific network, or could it work on the standard bridge network? Does the other Seafile container typically run on this network?

Reply:
It just need to be able reach the seafile server container and the DB. Thats why is the easiest and most secure way for all those containers to be in the same docker network and just proxy the web interface port and the notification port to the host, and all the other ports stay inside the docker network.

Reply:
This needs documenting in the overview